Makefile

```diff
@@ -12,6 +12,7 @@ install:
 	go generate ./...
 	go install -v ./...
 	go test -i -v ./...
+	git diff --exit-code objects/type_string.go
```

Comment:
That's will make development impossible. 😞 I _think_ that should be just `git diff --exit-code` (for the whole tree and in Travis CI configuration). Only CI should check that, but not the developer.